lib/spec/openapi/utils.js

```diff
@@ -91,6 +96,18 @@ function transformDefsToComponents (jsonSchema) {
   return jsonSchema
 }
 
+function convertExamplesArrayToObject (examples) {
+  return examples.reduce((examplesObject, example, index) => {
+    if (typeof example === 'object') {
+      examplesObject['key' + index] = { value: example }
```

Comment:
Let change the key to example to be more appropriate.
```suggestion
      examplesObject['example' + index] = { value: example }
```

Reply:
Sure, great idea, it got much better, just added these changes